### PR TITLE
Lecture at university of hamburg

### PR DESCRIPTION
**Pull request by RicardoUsbeck**
That is, what I would be going to teach. Thanks for all your material Olaf and Pierre-Antoine. And thanks to Longquan (@LongquanDoesNLP) for preparing most of the material.

### CODE REVIEW

File: lecture/RDF_star-SPARQL_star.tex [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+     \item Use Cases for justification are collected \url{https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html}
+     \item Still an active field of discussion \url{https://lists.w3.org/Archives/Public/public-rdf-star/2021Dec/0001.html}
+     \item (Strongly disputed) Use case by Amazon \url{https://lists.w3.org/Archives/Public/public-rdf-star/2021Dec/att-0001/rdf-star-neptune-use-cases-20211202.pdf} $\rightarrow$ We are still not able to model every real-world use case with satisfaction
+ \end{itemize}
+
+\end{frame}
+
+\begin{frame}{Summary}{Outlook}
+	\begin{itemize}
+	    \item 2nd community report underway \url{https://w3c.github.io/rdf-star/cg-spec/editors_draft.html}
```
Comment (pchampin) [inline lecture/RDF_star-SPARQL_star.tex]: actually, that will be the 4th one :)

**Comment by pchampin** [comment]
Regarding the "Singleton property" examples, it is weird to name the singleton property "type1" (I suspect this is a reminiscence from the Lotico examples). The suggested practice is to name the SP after the original property, so in this case, that would be "jobTitle1".

Of course, that does not make the example invalid -- the actual IRI of the SP is not relevant. But that makes it slightly confusing, IMO.

File: lecture/RDF_star-SPARQL_star.tex (hunk 1 above)
Comment (RicardoUsbeck) [reply lecture/RDF_star-SPARQL_star.tex]: Changed :)

**Comment by RicardoUsbeck** [comment]
> 

@LongquanDoesNLP could you redo the graphic please?

**Comment by LongquanJiang** [comment]
Ok

**Pull request merged**